styles/brand-tokens.css

```
@@ -1,19 +1,66 @@
-/* Kaiord Brand Tokens — shared across landing and editor */
+/* Kaiord Brand Tokens — shared across landing, docs, and editor.
+   Changes here affect all three surfaces.
+
+   NOTE: All values are dark-mode-specific.
+   When adding light mode, move these under .dark and add :root light defaults. */
+
+@font-face {
+  font-family: "Inter";
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)